tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,56 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+var =
+"Hello World!"
+"""
+
+
+bad_file_unexpected_comma = """
+module UnexpectedComma (..) where
+va,r =
+"Hello World!"
+"""
+
+
+good_file_no_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+100
+"""
+
+
+bad_file_unexpected_end_of_input = """
+module UnexpectedEndOfInput (..) where
+var =
+"""
+
+
+good_file_not_empty = """
```

Comment:
According to http://elm-lang.org/try , this doesnt compile.
It would need to be
```
foo =
     100
```
But that is testing too many things.  Need minimal tests; the smallest 'not_empty' using as few language features as possible.

Then add more good sample files for more elaborate language features.